ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6368,3 +6368,84 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 24h
+  labels:
```

Comment:
Can you give this a job-env label with the value "metal-ipi"? 

So we can enable cluster-bot for this: https://github.com/openshift/ci-chat-bot/blob/master/prow.go#L38-L40

Reply:
It would probably be just `metal` (the existing one should be `metal-upi`).  IPI is our preferred.  But you can do in follow up.